Datadog.Trace.sln

```diff
@@ -524,14 +524,14 @@ Global
 		Release|Any CPU = Release|Any CPU
```

Comment:
has this accidentally been comitted or are they necessary changes 🤔 ?

Reply:
The solution is broken for VS. It makes this changes when reparing it. I guess it is irrelevant for the build and only affects VS

Reply:
I recommend reverting this. Many of these changes are included in https://github.com/DataDog/dd-trace-dotnet/pull/4314